wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <!--create a leader board that shows top scores-->
+    <v-card>
+        <v-card-title class = "text-center">Leader Board</v-card-title>
+        <v-table>
+            <thead>
+                <tr>
+                    <th>User Name</th>
+                    <th>Average Attempts</th>
+                    <th>Game Count</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="score in scoresToDisplay" :key="score.name">
+                    <td v-if="score.name">{{ score.name }}</td>
+                    <td v-if="score.averageAttempts">{{ score.averageAttempts }}</td>
+                    <td v-if="score.gameCount">{{ score.gameCount }}</td>
```

Comment:
```suggestion
          <td>{{ score.name }}</td>
          <td>{{ score.averageAttempts }}</td>
          <td>{{ score.gameCount }}</td>
```
I don't see the purpose of doing `v-if` here. This would check if there were undefined values within the sores you retrived from the server but we are making sure that there can't be null/undefined values. I could maybe see this being helpful if you wanted to display a message if no one was in the leaderboard

Reply:
Yeah, I honestly have no clue why those are there. We'll fix that ASAP.

Reply:
the if is making sure that the item is there before displaying it

Reply:
Johanne makes a valid point here. `scoreToDisplay` won't be iterated if it's empty, and also if those values are null or undefined they will just interpolate to an empty string.